instrumentation/opentelemetry-instrumentation-aio-pika/tests/test_publish_decorator.py

```diff
@@ -54,7 +55,10 @@ def setUp(self):
         asyncio.set_event_loop(self.loop)
 
     def test_get_publish_span(self):
-        exchange = Exchange(CONNECTION, CHANNEL, EXCHANGE_NAME)
+        if version.parse(__version__) >= version.parse("8.0.0"):
```

Comment:
The commit for this breaking change - https://github.com/mosquito/aio-pika/commit/c2ee4be6e4f254e0d36c99f5bcd906cf31fa8233